src/main/resources/dashboard/repo.hbs

```diff
@@ -35,13 +35,15 @@ repo:
       - download
 {{/if}}</textarea>
     <input name="repo" type="hidden" value="{{name}}"/>
+    <input name="method" type="hidden" value="update"/>
```

Comment:
Term `method` is confusing here: method in HTTP protocol is a part of request line and it also can have value `delete`, so I'd change it to `action` or similar